### PR TITLE
chore: fix doc comments

### PR DESCRIPTION
**Pull request by revolunet**
Just some doc comments fixes

### CODE REVIEW

**Review by anoopshrma — APPROVED** [review]
Looks good! , Just need to check on the lint issue then we can get it merged

File: llama-index-integrations/readers/llama-index-readers-file/llama_index/readers/file/markdown/base.py [hunk 1]
```diff
@@ -71,12 +71,12 @@ def markdown_to_tups(self, markdown_text: str) -> List[Tuple[Optional[str], str]
         return markdown_tups
 
     def remove_images(self, content: str) -> str:
-        """Get a dictionary of a markdown file from its path."""
+        """Remove images in markdown content"""
```
Comment (logan-markewich) [inline llama-index-integrations/readers/llama-index-readers-file/llama_index/readers/file/markdown/base.py:74]: This ending with a period will fix linting (same with the other one)

Comment (revolunet) [reply llama-index-integrations/readers/llama-index-readers-file/llama_index/readers/file/markdown/base.py]: oops, fixed

**Comment by anoopshrma** [comment]
Linting passed! 
Thanks for the prompt action @revolunet! 
Merging it now

**Pull request merged**